OpenSim/Simulation/OpenSense/InverseKinematicsStudy.h

```diff
@@ -9,6 +9,9 @@
  * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
  * through the Warrior Web program.                                           *
  *                                                                            *
+ * Distribution Statement A – Approved for Public Release; Distribution is    *
+ * Unlimited.                                                                 *
+ *                                                                            *
  * InverseKinematicsStudy by AMJR consulting under a contract and in a        *
```

Comment:
It seems like there are some words missing here. Maybe "InverseKinematicsStudy was developed in part by AMJR consulting" , since I believe there has been additional work on this part of the code as part of work at Stanford.